src/init.ts

```diff
@@ -378,7 +383,7 @@ export class NightwatchInit {
     const tplData = fs.readFileSync(templateFile).toString();
 
     let rendered = ejs.render(tplData, {
-      plugins: false,
+      plugins: JSON.stringify(plugins).replace(/"/g, '\'').replace(/\\\\/g, '/'),
```

Comment:
```suggestion
      plugins: JSON.stringify(plugins).replace(/"/g, '\''),
```
as plugins don't have path (this was added to replace `\` in windows path to `/`)